contracts/tasks/whitelist.ts

```diff
@@ -0,0 +1,30 @@
+// Define the task with the name 'whitelist-address'
+task("whitelist", "Whitelists an address in the Oracle contract")
```

Comment:
Maybe here can just have something like
`export const whitelistTask = task("...`
Also remove module.exports

And then in hardhatconfig can use import instead of require
import {whitelistTask} from "./......."